packages/fetchai/connections/ledger/ledger_dispatcher.py

```diff
@@ -194,7 +194,7 @@ def get_transaction_receipt(
         while (
             not is_settled
             and attempts < self.MAX_ATTEMPTS
-            and self.connection_status.is_connected
+            and self.connection_state.get() == ConnectionStates.connected
```

Comment:
I think you have a property for this now; is_connected

Reply:
I pass state not connection, there is no is connected property